### PR TITLE
doc: update faq.md

### PR DESCRIPTION
**Pull request by diabhey**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -2,5 +2,44 @@
 slug: faq/
 title: FAQ
 ---
+## Setup and Installation
 
-Coming Soon: Frequently Asked Questions.
+**What are the K8s cluster requirements to install Otomi?**
+
+Otomi requires a running Kubernetes cluster of version `1.18` up to `1.21` with at least `3 worker nodes` (using General Purpose instance types with at least `4 vCPU` and `12 GiB memory`) in AWS, Azure, Google Cloud Platform.
```
Comment (Morriz) [inline docs/faq.md:9]: yeah, just say 12 vCPU as number of nodes is not important

File: docs/faq.md [hunk 2]
```diff
@@ -2,5 +2,44 @@
...
+
+No. The [otomi-quickstart](https://github.com/redkubes/quickstart) uses terraform to spin up a three-node Kubernetes cluster and install Otomi.
+One can spin up a K8s cluster and install Otomi via helm chart.
+
+Check [chart-install](https://otomi.io/docs/installation/chart) for more details.
+
+---
+**Where can we find all possible configuration parameters for Otomi install?**
+
+The [otomi-core/values-schema.yaml](https://github.com/redkubes/otomi-core/blob/master/values-schema.yaml) contains all the possible Otomi chart install configuration parameters.
```
Comment (Morriz) [inline docs/faq.md:26]: please refer to the `.json` schema delivered in the chart first, and you could mention that it is derived from that yaml file.

File: docs/faq.md [hunk 3]
```diff
@@ -2,5 +2,44 @@
...
+
+---
+
+## Otomi Configuration
+
+**Can we disable applications such as `Vault` and `Harbor` if we want to?**
+
+Yes, it is possible to disable applications. One can do so in the [values.yaml](https://github.com/redkubes/otomi-core/blob/master/chart/otomi/values.yaml) that is used in the Otomi chart install.
```
Comment (Morriz) [inline docs/faq.md:34]: Please state clearly that certain apps can't be disabled. The schema will be leading in that regard

**Review by Morriz — CHANGES_REQUESTED** [review]

File: docs/faq.md (hunk 2 above)
Comment (diabhey) [reply docs/faq.md:26]: Is it not easy to verify from the `values-schema.yaml` itself before installing?. Where is this .json file?

Comment (Morriz) [reply docs/faq.md:26]: In the chart root

**Comment by diabhey** [comment]
Closing as per Sander's request

**Comment by Morriz** [comment]
@diabhey what did Sander say about it?

**Comment by diabhey** [comment]
@Morriz: I shall discuss it offline.

**Pull request closed without merging**